src/alembic/versions/20251031_0808_f109995eb51e_replace_parsed_probed_with_media_.py

```
@@ -0,0 +1,263 @@
+    columns = [col["name"] for col in inspector.get_columns(actual_table_name)]
+    has_parsed_data = "parsed_data" in columns
+    has_probed_data = "probed_data" in columns
+
+    if not has_parsed_data and not has_probed_data:
+        # Old columns don't exist - nothing to migrate
+        return
+
+    # Get all MediaEntry rows with parsed_data or probed_data
+    result = connection.execute(
+        sa.text(
+            f'SELECT id, original_filename, parsed_data, probed_data FROM "{actual_table_name}" WHERE parsed_data IS NOT NULL OR probed_data IS NOT NULL'
```

Ruff (0.14.3)

Possible SQL injection vector through string-based query construction

(S608)